functional-samples/cookbook.file_handlers/README.md

```diff
@@ -0,0 +1,25 @@
+# Cookbook - File Handling API
+
+This sample demonstrates how to use the File Handling API in an extension.
+
+## Overview
+
+On ChromeOS only, extensions can use the `file_handlers` manifest key to
+register as a file handler for particular file types. This behaves in the same
+way as the
+[equivalent API](https://developer.chrome.com/articles/file-handling/) in web
+applications.
```

Comment:
```suggestion
register as a file handler for particular file types. This behaves in a similar way to the
[equivalent key](https://developer.mozilla.org/docs/Web/Manifest/file_handlers) in web
applications. As with web applications, you use the [Launch Handler API](https://developer.mozilla.org/en-US/docs/Web/API/Launch_Handler_API) to open and process a file.
```

Reply:
>behaves in a similar way

I'm hedging here because I don't think we yet support all the functionality that's avaialable through the web platform API.